### PR TITLE
Centralized database

### LINKED ISSUE #6: Peer finding(later migrate to blind-find)

### LINKED ISSUE #7: Advertisement(Later migrate to decentralized storage)

### LINKED ISSUE #17: ERC20 Token Listing

### PR DESCRIPTION
**Pull request by tkmct**
use centralized db for advertisement and peer info
close #6, close #7, close #17

### CODE REVIEW

File: src/utils/string.ts [hunk 1]
```diff
@@ -9,3 +9,16 @@ export function truncateString(str: string, num: number): string {
...
+
+export function trimLeadingZeroesHex(hex: string): string {
+  const numPart = hex.substring(2)
+  const match = numPart.match(/^0+/)
+  const leadingZeroes = match ? match[0].length : 0
+
+  return `0x${numPart.substring(leadingZeroes)}`
+}
+
+export function padAddress(hex: string): string {
```
Comment (wanseob) [inline src/utils/string.ts:21]:
```
import { utils } from 'ethers';

utils.hexZeroPad('0xabcd', 20); // should return a same result with this padAddress func
```

File: src/utils/string.ts [hunk 2]
```diff
@@ -9,3 +9,16 @@ export function truncateString(str: string, num: number): string {
 export function shortAddressString(address: string): string {
   return `${address.slice(0, 6)}...${address.slice(address.length - 4)}`
 }
+
+export function trimLeadingZeroesHex(hex: string): string {
```
Comment (wanseob) [inline src/utils/string.ts:13]:
```typescript
import { utils } from 'ethers';

utils.hexStripZeros('0x0000abcd'); // should return a same result with this function
```

File: src/hooks/zkopru.ts [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { useMutation } from 'react-query'
 import { useWeb3React } from '@web3-react/core'
 import { sha512_256 } from 'js-sha512'
 import { fromWei } from '../utils/wei'
```
Comment (wanseob) [inline src/hooks/zkopru.ts:5]: corresponding utility functions in ethers.js are

fromWei => ethers.utils.formatEther
toWei => ethers.utils.parseEther

**Review by wanseob — COMMENTED** [review]
How about removing this function and using the ethers library instead? (BTW I also think that zkopru core package should use ethers library too)

Other changes look good to me ;)

**Comment by tkmct** [comment]
Thank you for the comments. I'll fix and merge

**Pull request merged**